### DISCUSSION TITLE
Email not working

### DISCUSSION THREAD

**Discussion by Sneeex** [requarks/wiki#4957] [Error / Bug Report]
Hey,
I tried using the email feature of Wikijs but somehow it isn't working, i tried the same configuration [here](https://www.smtper.net/) and it worked. Any ideas why? I don't get any errors when using the "send test mail" button.
I'm using the latest version wiki js (Wiki.js 2.5.274) and these are my mail settings: 
<img width="725" alt="image" src="https://user-images.githubusercontent.com/65130981/152149391-1bcf2429-d1f5-49ec-96c5-5f66b8a915f0.png">
Is there any log except the normal process log?

**Comment by Phil-hostwiki** [comment]
Try turning off "Secure (TLS)" and "Verify SSL Certificate".
